pkgs/applications/virtualization/singularity/packages.nix

```diff
@@ -9,14 +9,14 @@ let
     callPackage
       (import ./generic.nix rec {
         pname = "apptainer";
-        version = "1.3.3";
+        version = "1.3.4";
         projectName = "apptainer";
 
         src = fetchFromGitHub {
           owner = "apptainer";
           repo = "apptainer";
           rev = "refs/tags/v${version}";
-          hash = "sha256-xQZCQa9z1aJ2tVtxMlwcNhlm0EV/nn8OnbfaVZRm4JI=";
+          hash = "sha256-eByF0OpL1OKGq0wY7kw8Sv9sZuVE0K3TGIm4Chk9PC4=";
         };
 
         # Update by running
```

Comment:
The hash stayed the same?

Reply:
What do you mean ? I have changed it, no ?

I have double checked right now and both hashes seem correct.

Reply:
Yes. This can be verified by running `nix-prefetch -E "{ sha256 }: ((import ./. { }).apptainer.overrideAttrs { vendorHash = sha256; }).goModules"` at the Nixpkgs project root.

Reply:
> What do you mean?

@GaetanLepage I guess Sandro is talking about the `vendorHash`.

Reply:
Oh sure, I actually checked both hashes... but i hadn't push for all this time X)

Reply:
Ok, forget about my previous message. The hash has actually not changed between the two releases.